src/main/java/com/thirdcc/webapp/service/dto/EventDTO.java

```diff
@@ -32,6 +32,10 @@
 
     private EventStatus status;
 
+    private Long imageStorageId;
```

Comment:
the imageStorageId looks like redundant with imageStorageDTO

Reply:
imageStorageId will be save into DB
ImageStorageDTO is used when we response the EventDTO with Image Info